connector_importer/components/importer.py

```diff
@@ -363,9 +370,13 @@ def run(self, record, is_last_importer=True, **kw):
             ]
         ).format(**counters)
         self.tracker._log(msg)
+        self._set_shared_storage_values(self.tracker.chunk_report)
         self._trigger_finish_events(record, is_last_importer=is_last_importer)
         return counters
 
+    def _set_shared_storage_values(self, chunk_report):
```

Comment:
this is not setting anything. I'd prefer to keep it generic and rename it to `_finished_chunk_hook()`.
Just call `self._finished_chunk_hook()`, no need to pass the chunk_report as you have it in `self.tracker`.
Or maybe just pass the last line and odoo record if any.

Reply:
Actually: any reason why you cannot use the event hook instead?